portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includePage.vm

```diff
@@ -28,7 +28,7 @@
 #else
     #set( $numberTools = $pageColumn0Tools.size() )
 #end
-<div class="portal-main-container container-fluid mt-2">
+<div class="portal-main-container container-fluid mt-2 d-flex flex-column">
```

Comment:
The issue is specific to the gradebook table, so changes should be made in gradebook-tool. I don’t think we need to update the entire portal.

Reply:
The portal-main-container changes were a part of SAK-50454 that were not merged in 23.x. Hence, I added it here as well and made changes for the gradebook tool specifically (overflow-x: hidden in gradebookGradesToolbar).

Reply:
Thanks for the explanation, @i-a-m-s-k! It looks like [SAK-50454](https://sakaiproject.atlassian.net/browse/SAK-50454) may be merged into 23.x soon, which could lead to a conflict. I'll check in with @ern on this.